drbd/templates/global_common.j2

```diff
@@ -81,6 +81,15 @@ common {
 {%- if multi_primaries %}
                 allow-two-primaries {{ multi_primaries }};
 {%- endif %}
+{%- if max_buffers %}
```

Comment:
Back again with the defaults XD
I think it would be better if we had actual default values and set them instead of using these `if` statements

Reply:
I mean `max-buffers` is not configured by default. (Refer to `defaults.yaml`)
The default value in `global_confs.sls` is just an example.